src/domain/Phone/WebRTCPhone.js

```diff
@@ -841,6 +841,12 @@ export default class WebRTCPhone extends Emitter implements Phone {
       return Promise.resolve(null);
     }
 
+    const sipSessionId = this.getSipSessionIdFromCallSession(callSession);
+    if (sipSessionId && !(sipSessionId in this.callSessions)) {
+      logger.warn('Call session already ended can\'t accept it.', { sipSessionId });
```

Comment:
Edo: Are we sure it's ended? I'd lean towards a statement of fact (i.e. no callsession related to `sipSessionId`, bailing); this way, we avoid misinterpretation.